packages/accounts-base/accounts_server.js

```diff
@@ -402,6 +402,8 @@ export class AccountsServer extends AccountsCommon {
     if (! stampedLoginToken) {
       stampedLoginToken = this._generateStampedLoginToken();
       this._insertLoginToken(userId, stampedLoginToken);
+    } else {
+      this.users.update(userId, { $set: { lastLoginAt: new Date() } });
```

Comment:
This one gets triggered when user resumes their connection (i.e. refreshes the window or re-opens the page after a while when the login token is still present).

Reply:
I'm thinking this one could be potentially hidden behind a settings flag so that it doesn't get set if the site doesn't want to consider resumption a login event.